articles/azure/azs-how-config-vm-update-management.md

```diff
@@ -0,0 +1,196 @@
+
+    - Click the newly create workspace. On the *new* blade, under settings, select *Advanced settings*, then select *Windows* or *Linux* servers depending on the *VM Type* you wish to add analytics too. 
+    Note down the **Workspace ID** and **Primary Key** values.
+
+        ![Log Analytics workspace advanced settings](images\azs-browser-log-analytics-workspace-advanced-settings.PNG)
+
+    - Within *Advanced settings*, Select the **Data** blade, and click *`VM Type` Performance Counters.*
+
+    - Ensure all *counters* are selected and click **Add the selected performance counters**.
+
+    - Click **Save**.
+
+        ![Performance Counters](images/azs-browser-example-performance-counters.PNG)
+
+        > [!NOTE]
+        > For Linux Performance Counters, ensure **Apply below configuration to my machines** is selected.
+
+3. Create and configure an *Automation Account*:
+
+    - Click *+ Create a resource*.
+
+    - Search `automation` and click **Create**.
+
+        ![automation account](images\azs-browser-search-automation-account.PNG)
+
+    - Choose a *Name*, *Subscription*, *Resource Group*, *Location* and ensure *Create Azure Run As Account* is set to **Yes**.
+
+    - Click **Create**.
+
+        ![Add automation account](images/azs-browser-add-automation-account.PNG)
+
+        > [!NOTE]
+        > The error "Azure Classic Run As account creation error" may occur. This **DOES NOT** effect the process.
+
+    - Once deployed, navigate to the *Automation Account*. In the *new* blade, Under *Configuration Management*, select the solution to enable (Inventory & Change Tracking or Update management).
+
+    - Select the *Log Analytics workspace* to link the automation account too.
+
+    - Click **Enable**.
+
+    - Wait for the deployment to complete.
+
+    - Repeat the two previous steps for all two solutions (Inventory & Change Tracking and Update management).
+
+        > [!TIP]
+        > Leave the public azure portal open; it is still needed for later.
+
+4. Login to the Azure Stack portal:
+
+    <https://portal.frn00006.azure.ukcloud.com>
+
+5. Navigate to the VM which you wish to enable **Update & Management** on and under settings, select the *extensions* blade.
+
+    > [!WARNING]
+    > For any monitoring to work correctly. The VM **must** have HTTPS (Port 443) enabled in the **Network Security Group** rules.
+
+6. Click **Add** at the top and select the extension `Azure Monitor, Update and Configuration Management`; then click **Create**.
+
+    ![VM enable update management](images/azs-browser-log-analytics-enable-update-management.PNG)
+
+    - Provide the extension with the **Workspace ID** and **Primary Key** values (noted down previously).
+
+    - Repeat this step for the `Azure Monitor Dependency Agent`.
+
+7. Once the deployment is finished. Head back to the public Azure portal, navigate to the *Log Analytics workspace* -> *Advanced settings* -> *`VM Type` Servers*.
```

Comment:
comma instead of full stop:
Once the deployment is finished, head back to the public Azure portal and navigate to ...